homework.md

```diff
@@ -4,38 +4,38 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
```

Comment:
Hi @Domenico-cyf , when I did this question I did not elaborate on the answer much, maybe you can do the same here as well.

Reply:
Yes, I will do the same and leave it, there is no much to elaborate.

Reply:
No sorry what I meant is maybe explain how CSS makes the website better, do you mean look better in terms of the stlying (like fonts, colors, layouts) OR look better in terms of preformance which is something JS related.